examples/guidelines-snap-tutorials.md

```diff
@@ -145,190 +255,384 @@ Survey
 
-In particular, to avoid a teacher/student relationship, do not separate questions and answers (apart from cliffhangers as previously stated, but the question is more rhetorical in that case!) with actionable attendee’s expectation in different steps. The question/answer (command line or code snippet) should be on the same page. That way, the answer is always in the context of the given question.
+Avoid repetitive setups or installation phases, particularly if the tutorial isn’t a beginner one. Beginner tutorials should contain a setup phase, other more advanced tutorials should reference other beginner tutorials as prerequisites.
 
-#### Do not repeat the setup/install phase for each tutorial
+### Command line snippets
+
+Inline commands are styled with single backticks :
+
+For example:
+
+```markdown
+`foo/something --bar`
+```
 
-Avoid repetitive setups or installation phases, particularly if the tutorial isn’t a beginner one. Beginner tutorials should contain a setup phase, other more advanced tutorials should reference as prerequisites other beginner tutorials directly.
+Whic renders as `foo/something --bar`.
 
-#### Command lines
 
-Inline commands are styled simply by backticks :`foo/something --bar`.
+For longer example code we expect people to type in, we do not use the command prompt at the beginning of each line, and we separate the command from the output. This makes the command and outputs clearer and also easier to copy and paste.
 
+#### Example
 
-For actual example code we expect people to type in, we do not use the command prompt, and we separate the command from the output. This makes it clearer and unambiguous and also easier to copy and paste. For example:
+"
+Run the following command:
 
-Run the following...
 ```bash
 cat my_file
 ```
-… to see the contents of the file:
+
+This will display the content of the file:
 
 ```bash
 Awesome my_file content
 on multiple lines
 ```
-...before removing it with the command:
+"
 
-```bash
-rm my_file
-```
+Finally, the code blocks (commands and code to write) of the tutorial should be self-sufficient. It means that we don't expect people to write code or run commands outside of what we expose in code blocks, and only typing and executing their content should lead them to the desired state.
+
+You should now have all the cards in hand to provide a rewarding learning experience! It's time we look at how to structure the final page of a tutorial: the "rewarding" step.
+
+## Rewarding your readers
+Duration: 2:00
+
+On the last page of a tutorial, the project is finished and you remember attendees they have reached their goal and have grown some new skills.
+
+Take one or two paragraphs to remind them of what they went through and what they have achieved. That way, attendees can reflect upon the various steps and this new acquired knowledge. Ensure you use a friendly title for this final step. “That’s all folks!” is used in some other tutorials.
```

Comment:
and `their newly` acquired knowledge

Reply:
I'm not sure we should be recommending writers end with `That's all folks!` It's like comic sans in word form.

Reply:
What could it be? Something cheerful, and maybe slightly informal. For now, I'm changing to: Ensure you use a friendly title for this final step.